addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15898,7 +15898,7 @@ msgstr ""
 #. Description for video related setting #13457: vaapi sw filter
 #: system/settings/settings.xml
 msgctxt "#36433"
-msgid "If enabled VAAPI render method is prefered. This puts less load on the CPU but driver may hang!"
+msgid "If enabled VAAPI render method is preferred. This puts less load on the CPU but driver may hang!"
```

Comment:
This sentence is a bit strange. `If enabled, the VAAPI render method will be preferred. This puts less load on the CPU, but may cause the driver to hang!`